.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: build
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build the article PDF
+    concurrency: showyourwork-${{ github.ref }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install TinyTex for matplotlib LaTeX rendering
```

Comment:
@kazewong I'm explicitly installing TeX now, so the matplotlib stuff should work (still untested though)